trace/trace.go

```diff
@@ -42,8 +36,8 @@ type key string
 
 const traceKey key = "trace"
 
-// Service is our service name and should be set exactly once,
-// at startup
+// Service is our service name and must be set exactly once,
+// at startup.
```

Comment:
`must be set exactly once, by the main package. It is recommended to set this value in an init() function or at the beginning of the main() function`.